Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN npm install --legacy-peer-deps --production
 # Copy the rest of the application code
 COPY . .
 
+COPY ./example.env ./.env
```

Comment:
We will end up with copying/overwriting `.env`(which has API keys and Tokens) with placeholder values on every deployment so this should not be added